services/tenant-ui/frontend/src/components/layout/AppLayout.vue

```diff
@@ -41,7 +41,7 @@ body {
   display: flex;
   flex-direction: row;
   align-items: flex-start;
-  height: 100vh;
+  min-height: 100vh;
```

Comment:
I just pushed these to get the sidebar color to stretch down when the page expands (was constraining it to only fill the page height when it should fill AT LEAST the page height)

Reply:
👍